release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -456,18 +456,27 @@ Enabling this service monitor eliminates the possibility that two queries sent a
 ==== Update to Alertmanager configuration for additional secret keys
 With this release, if you configure an Alertmanager secret to hold additional keys and if the Alertmanager configuration references these keys as files (such as templates, TLS certificates, or tokens), your configuration settings must point to these keys by using an absolute path rather than a relative path.
 These keys are available under the `/etc/alertmanager/config` directory.
-In earlier releases of {product-title}, you could use relative paths in your configuration to point to these keys because the Alertmanager configuration file was located in the same directory as the keys. 
+In earlier releases of {product-title}, you could use relative paths in your configuration to point to these keys because the Alertmanager configuration file was located in the same directory as the keys.
 
 [IMPORTANT]
 ====
-If you are upgrading to {product-title} {product-version} and have specified relative paths for additional Alertmanager secret keys that are referenced as files, you must change these relative paths to absolute paths in your Alertmanager configuration. 
+If you are upgrading to {product-title} {product-version} and have specified relative paths for additional Alertmanager secret keys that are referenced as files, you must change these relative paths to absolute paths in your Alertmanager configuration.
 Otherwise, alert receivers that use the files will fail to deliver notifications.
 ====
 
 
 [id="ocp-4-12-scalability-and-performance"]
 === Scalability and performance
 
+==== Disabling realtime using workload hints removes Receive Packet Steering from the cluster
+By default, a systemd service sets a Receive Packet Steering (RPS) mask for virtual network interfaces in the cluster. The RPS mask routes interrupt requests from virtual network interfaces according to the list of reserved CPUs defined in the performance profile. A crio hook script also sets an RPS mask for all virtual network devices at the container level.
```

Comment:
```suggestion
At the cluster level by default, a systemd service sets a Receive Packet Steering (RPS) mask for virtual network interfaces. The RPS mask routes interrupt requests from virtual network interfaces according to the list of reserved CPUs defined in the performance profile. At the container level, a crio hook script also sets an RPS mask for all virtual network devices.
```

Reply:
I thought this might help clarify the two different levels RPS is being set at. Maybe it helps?